static/js/message_edit.js

```diff
@@ -274,16 +277,25 @@ function edit_message(row, raw_content) {
         file_upload_enabled: file_upload_enabled,
         show_video_chat_button: show_video_chat_button,
         minutes_to_edit: Math.floor(page_params.realm_message_content_edit_limit_seconds / 60),
+        show_edit_stream: show_edit_stream,
+        available_streams: available_streams,
     }));
 
     const edit_obj = {form: form, raw_content: raw_content};
     currently_editing_messages.set(message.id, edit_obj);
     current_msg_list.show_edit_message(row, edit_obj);
 
+    if (show_edit_stream) {
+        $("#select_stream_id_" + message.id)
+            .find("option[value=" + message.stream_id + "]")
+            .prop("selected", true);
+    }
```

Comment:
Should this bit of logic be in the template instead?

Reply:
No longer required, I selected a default value in handlebar for this.